### PR TITLE
reset gradle & jdk11 dependency

### PR DESCRIPTION
**Pull request by bofeng-song**

### CODE REVIEW

File: native/cocos/platform/android/libgame-sdk/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.cocos.gamesdk">
+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```
Comment (minggo) [inline native/cocos/platform/android/libgame-sdk/src/main/AndroidManifest.xml]: Why need network permission?

**Comment by minggo** [comment]
Why need both source codes and jar?

File: native/cocos/platform/android/libgame-sdk/src/main/AndroidManifest.xml (hunk 1 above)
Comment (bofeng-song) [reply native/cocos/platform/android/libgame-sdk/src/main/AndroidManifest.xml]: Copy from libcocos, it can be remove

**Comment by bofeng-song** [comment]
> Why need both source codes and jar?

We've change some code, source code only store. App uses jar only

**Review by PatriceJiang — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**